### ISSUE TITLE
The Meta Profile export is broken on the standard Stream Deck

### ISSUE THREAD

**Issue by mitsuo-makuda** [DavidGretzschel/StreamDeckXL_Meta-Profile#2]

**Comment by DavidGretzschel** [comment]
This is a profile for a StreamDeck XL. The whole meta-idea would require a different layout for a normal StreamDeck. Wish I would have added some pictures to make this a bit clearer, but atm I don't actually own any StreamDecks of any kind.

**Comment by mitsuo-makuda** [comment]
Right, do you have any plans to port it to the other sizes out of curiousity?

**Comment by DavidGretzschel** [comment]
No. Wouldn't even know how, since I don't own a StreamDeck, anymore.

But doing a Meta-Layout is not difficult to do. (just a little tedious, hence this repo)
Just figure out how to split off all possible shortcuts into 15 folders of 14 icons each in a sensible grouping. Except..... there's already more than 14 letters of the alphabet, so whatever you do, it's not gonna be terribly pretty.
With the 32 buttons of the XL you could do all aphabet + all numeral shortcuts per modifier group in one folder and everything else in another.
With the normal StreamDeck, you'd probably want two Layouts or a folder structure that's two levels deep.
I think I'd prefer two connected Layouts (with a button switching between them). One for the groups with shift, one for the groups without shift.
Though I don't know what's easy or possible to do with the latest StreamDeck software.
